lib/fedora_migrate/repository_migrator.rb

```diff
@@ -52,6 +33,22 @@ def get_source_objects
 
     private
 
+    def migrate_object source
+      Logger.info "Migrating source object #{source.pid}"
+      FedoraMigrate::ObjectMover.new(source, nil, options).migrate
+    rescue StandardError => e
+      Logger.warn "#{source.pid} failed.\n#{error_message(e)}"
+      @failed = @failed + 1
+    end
+
+    def migrate_relationship source
```

Comment:
I appreciate the parallelism between these two migrate methods now. You can almost see a refactor coming along... :)